Task2_2.py

```diff
@@ -0,0 +1,38 @@
+import string
+import random
+
+# creating list of dicts
+list_of_dicts = []
+# set number of dicts
+for k in range(random.randint(2, 10)):
```

Comment:
1) You don't use 'k' anywhere inside loop, then omit it by _
2) Variable names should be **meaningful**, please no more i, j, k etc.

Reply:
Got it.

Reply:
Please remove all unneccassary imports next time